### DISCUSSION TITLE
Just wanted to say thanks.

### DISCUSSION THREAD

**Discussion by ratty222** [Lissy93/dashy#384] [General]
I've really enjoyed watching your work come together.  Thanks for sharing your efforts, and I'm looking forward to seeing where you take Dashy.  I've learned a lot just by watching your updates.

My dashboard is nowhere near complete, but here are the screenshots:

![dashboard-desktop](https://user-images.githubusercontent.com/92832598/147505084-539bbf85-801d-477d-a165-1f1fb046c595.png)
![dashboard-mobile](https://user-images.githubusercontent.com/92832598/147505087-472d627d-5131-4b47-8128-3a1c4ba326c0.png)

**Comment by lissy93** [comment]
Heya @ratty222 - Thank you so much, this means a lot, and I love hearing about how people are using Dashy :)
Your dashboard looks awesome, would you mind if I added it to the [Showcase](https://github.com/Lissy93/dashy/blob/master/docs/showcase.md) page?
Thanks again, and feel free to reach out if there's anything you need,
Happy holidays, Alicia :)

**Reply by ratty222** [reply]
Yes, feel free to use the screenshots for your showcase.  Thanks again for sharing your work!

**Comment by ratty222** [comment]
Polished and finished with new widgets. Thanks again for your work :))

![ratboard20220107](https://user-images.githubusercontent.com/92832598/148615324-630103ad-2e12-49ac-a01f-f9420d077601.png)
!

**Reply by lissy93** [reply]
It's beautiful!
Thank you for sharing, and thank you very very much for your sponsorship. That's super kind of you 💖 
If there's ever anything you need, feel free to reach out :)
